src/Helpers/note.helpers.ts

```diff
@@ -0,0 +1,84 @@
+import mongoose from 'mongoose';
```

Comment:
```suggestion
import {Types} from 'mongoose';
type ObjectId = Types.ObjectId
```
can you just import the Objectid instead of full mongoose and modify the usage as ObjectId instead of `mongoose.Types.OnjectId`